data/org.freedesktop.portal.GlobalShortcuts.xml

```diff
@@ -0,0 +1,165 @@
+      @short_description: GlobalShortcut portal backend interface
+
+      This this portal lets applications register global shortcuts so they can
+      act regardless of the system state upon an input event.
+
+      To use this, an app has to create a session under which its actions
+      will be collected. Then shortcuts can be bound (see Bind), listed (see ListShortcuts).
+      It's up to the portal implementation to perform the registration
+      and tracking of that shortcut throughout the session's lifespan.
+
+      Upon a shortcut trigger, Activated will be emitted until when it's not
+      triggered anymore, then Deactivated will be issued.
+  -->
+  <interface name="org.freedesktop.portal.GlobalShortcuts">
+        <!--
+        CreateSession:
+        @options: Vardict with optional further information
+        @handle: Object path for the #org.freedesktop.portal.Request object representing this call
+
+        Creates a global shortcuts session.
+
+        Supported keys in the @options vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>handle_token s</term>
+            <listitem><para>
+              A string that will be used as the last element of the @handle. Must be a valid
+              object path element. See the #org.freedesktop.portal.Request documentation for
+              more information about the @handle.
+            </para></listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>session_handle_token s</term>
+            <listitem><para>
+              A string that will be used as the last element of the session handle. Must be a valid
+              object path element. See the #org.freedesktop.portal.Session documentation for
+              more information about the session handle.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+
+        The following results get returned via the #org.freedesktop.portal.Request::Response signal:
+        <variablelist>
+          <varlistentry>
+            <term>session_handle o</term>
+            <listitem><para>
+              The session handle. An object path for the
+              #org.freedesktop.portal.Session object representing the created
+              session.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+    -->
+    <method name="CreateSession">
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="handle" direction="out"/>
+    </method>
+
+      <!--
+        ListShortcuts:
+        @session_handle: Object path for the #org.freedesktop.impl.portal.Session object representing the session
+        @options: Unused so far
+
+        List all the shortcuts with their invocation descriptions.
+
+        The following results get returned via the #org.freedesktop.portal.Request::Response signal:
+        <variablelist>
+          <varlistentry>
+            <term>shortcut_ids a{sv}</term>
```

Comment:
Why do we want to receive back the description of the shortcut?

Reply:
I don't think we want, this was just an example. Though, I'm considering the possibility of making `trigger` be an array of strings (and rename it to `triggers` of course), so that this portal can eventually deal with multiple bindings.

Reply:
This is somewhat the case now, please re-review